.gitignore

```diff
@@ -82,4 +82,7 @@ fsacpath.txt
 /out/
 
 build/bin
-build/obj
\ No newline at end of file
+build/obj
+
+# include vscode ui toolkit minified release file
+!release/toolkit.min.js
```

Comment:
super weird - can you add not a reference to the toolkit and let webpack pull it in (I'm just skimming their [getting started](https://github.com/microsoft/vscode-webview-ui-toolkit#getting-started) docs)

Reply:
I tried following the getting started from the toolkit but their getting started assumes the directory where you launch your extension is the root of the project (hence why it had node_modules available), in our case the _release_ directory is the root so we don't have access to node_modules, we'd likely need to do some webpack magic to pull this script at dev/build which was the part I couldn't figure out

Reply:
Gotcha. I'd also be ok longer term with removing the release split. It's just confusing IMO.